package.json

```diff
@@ -1,4 +1,6 @@
 {
+  "name": "o-element-visibility",
+  "version": "2.1.5",
```

Comment:
We don't version our components using npm as we do not release them onto the npm registry. What we have done for other components is give them a version number of `0.0.0` in `package.json`, could this also be done here? This change won't affect how you install the component as you are using git urls which look for tagged commits/branches, which is what we have.

Also, we are looking to deprecate this component in the near future as it can be replaced by the new IntersectionObserver API. For browsers which don't support this API natively, we serve a polyfill from https://polyfil.io.